app/controllers/users_controller.rb

```diff
@@ -1598,7 +1598,11 @@ def bookmarks
         end
       end
       format.ics do
-        @bookmark_reminders = Bookmark.where(user_id: user.id).where.not(reminder_at: nil).joins(:topic)
+        @bookmark_reminders = Bookmark.with_reminders
+          .where(user_id: user.id)
+          .joins(post: :topic)
```

Comment:
Just curious, do we need to join here? I think eager loading is sufficient for the use case here.